src/components/orders/FilledProgress/index.tsx

```diff
@@ -26,13 +33,71 @@ const Wrapper = styled.div`
 `
 
 export function FilledProgress(props: Props): JSX.Element {
-  const { percentage = '0' } = props
+  const {
+    order: {
+      filledAmount,
+      filledPercentage,
+      kind,
+      buyAmount,
+      sellAmount,
+      buyToken,
+      sellToken,
+      buyTokenAddress,
+      sellTokenAddress,
+    },
+  } = props
+
+  let mainToken
+  let mainAddress
+  let mainAmount
+  let swappedToken
+  let swappedAddress
+  let swappedAmount
+  let action
+
+  if (kind === 'sell') {
+    mainToken = sellToken
+    mainAddress = sellTokenAddress
+    mainAmount = sellAmount
+    swappedToken = buyToken
+    swappedAddress = buyTokenAddress
+    swappedAmount = buyAmount
+    action = 'sold'
+  } else {
+    mainToken = buyToken
+    mainAddress = buyTokenAddress
+    mainAmount = buyAmount
+    swappedToken = sellToken
+    swappedAddress = sellTokenAddress
+    swappedAmount = sellAmount
+    action = 'bought'
+  }
+
+  // In case the token object is empty, display the address
+  const mainSymbol = mainToken ? safeTokenName(mainToken) : mainAddress
+  const swappedSymbol = swappedToken ? safeTokenName(swappedToken) : swappedAddress
+  // In case the token object is empty, display the raw amount (`decimals || 0` part)
+  const formattedFilledAmount = formatSmart(filledAmount.toString(10), mainToken?.decimals || 0)
+  const formattedMainAmount = formatSmart(mainAmount.toString(10), mainToken?.decimals || 0)
+  const formattedSwappedAmount = formatSmart(swappedAmount.toString(10), swappedToken?.decimals || 0)
```

Comment:
Question for the reviewer: Should we have greater precision here?

Right now it'll show `< 0.001` for anything < `0.001`...

Since the amounts here are redundant with info elsewhere I chose not to change this initially.
Have nothing against making it more precise, though.

Reply:
Might indeed be relevant for certain tokens. Although you could opt to address this at a later point. Unless the level of efforts is low to support greater precision.

Reply:
This is quite easy do add. Just a matter of what should we default to

Reply:
i mean 0 seems low. considering it's sth like 18 (from token.decimals) else 0. Maybe do default 4 or something

Reply:
Default to `0` is only when the token is not set. So it "gracefully" handles when there's no token information and shows the raw amount.
This shouldn't happen anyway, as the parent component will not render when that's missing.
Added just for the sake of completeness. The alternative is to return `null` and not render the component at all.

Anyway, that's not what I was asking for. It's whether the format `decimals` (different than `precision`) should be higher.

Reply:
In general, for the explorer I don't like too much the `< 0.001` thing. Reason, is because it exposes the info from the protocol, as informative as possible. Even `0.001` can be worth something, if we talk about BTC it's almost 50$

If it's clearly redundant, maybe is fine. But only if it's very clear to the user.

Reply:
ook, will update in another PR

Reply:
https://github.com/gnosis/gp-ui/pull/280